velox/expression/CastExpr-inl.h

```diff
@@ -275,6 +275,30 @@ VectorPtr CastExpr::applyDecimalToIntegralCast(
   return result;
 }
 
+template <typename FromNativeType>
+VectorPtr CastExpr::applyDecimalToBooleanCast(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    const TypePtr& /*fromType*/,
+    const TypePtr& toType) {
+  VectorPtr result;
+  context.ensureWritable(rows, toType, result);
+  (*result).clearNulls(rows);
+  auto resultBuffer =
+      result->asUnchecked<FlatVector<bool>>()->mutableRawValues<uint64_t>();
+  const auto simpleInput = input.as<SimpleVector<FromNativeType>>();
```

Comment:
cc: @kagamiori:  Just want to validate that input will always be SimpleVector when casting between types.

Reply:
Actually, I just copy the code here. https://github.com/facebookincubator/velox/blob/main/velox/expression/CastExpr-inl.h#L247